test/test_wiki.rb

```diff
@@ -303,18 +303,41 @@
     assert @wiki.page("Saruman")
   end
 
-  test "write page is not allowed to overwrite file" do
+  test "write page is not allowed to overwrite pages by default" do
```

Comment:
Is there a possibility to run these tests against a wiki with `page_file_dir` set, just to test whether it works with that?